pkgs/by-name/fi/fish-lsp/package.nix

```diff
@@ -45,6 +45,12 @@ stdenv.mkDerivation rec {
     yarn --offline compile
   '';
 
+  # We do it in postPatch, since it needs to be fixed before buildPhase
+  postPatch = ''
+    substituteInPlace bin/fish-lsp \
+      --replace "#!/usr/bin/env node" "#!${lib.getExe nodejs}"
```

Comment:
```suggestion
      --replace-fail "#!/usr/bin/env node" "#!${lib.getExe nodejs}"
```

Reply:
Does adding `nodejs` in the buildinputs will not automatically change this automatically?